### ISSUE TITLE
Zatrzymywanie się oddtwarzania 

### ISSUE THREAD

**Issue by Vitara2023** [AndreyPavlenko/Fermata#507]
Podczas odtwarzania youtube na jednostce multimedialnej bosch Vitara przez headunit reloaded - lustro, zatrzymuje się obraz i zrywa połączenie lustrzane. Telefon samsung S23. Nie ma znaczenia, czy przez kabel USB, czy adapter wifi, to lustro telefonu z jednostką główną się rozłącza. Zdarza się też tak, że AA  rozłączy się i po chwili znów połączy. Nawiązywanie połączenia lustro podczas jazdy jest utrudnione. Udaje się po wielu próbach, bo każde kliknięcie SELF MODE rozłącza na chwilę AA. Kolejny problem, to rozłączanie AA w tych samych miejscach przejazdu na drodze.

**Comment by AndreyPavlenko** [comment]
Try disabling the power saving for the app in the system settings. Probably, the system kills the app to save the power.

**Comment by Vitara2023** [comment]
Tak, miałem zawsze ustawione bez ograniczeń dla fermata, AA, mapy i headunit.
